File: src/main/webapp/app/shared/model/custom/effectiveness-phase-per-criteria-data.ts
export interface IEffectivenessPhasePerCriteriaData {
    criteria?: any;
    sumValue?: number;
}

export class EffectivenessPhasePerCriteriaData implements IEffectivenessPhasePerCriteriaData {
    constructor(
        public criteria?: any,
        public sumValue?: number
    ) {
    }
}

File: src/main/java/com/marineindustryproj/service/dto/customs/EffectivenessPhasePerCriteriaData.java
package com.marineindustryproj.service.dto.customs;

import java.io.Serializable;

public class EffectivenessPhasePerCriteriaData  {

    private Object criteria;

    private Long sumValue;

    public Long getSumValue() {
        return sumValue;
    }

    public void setSumValue(Long sumValue) {
        this.sumValue = sumValue;
    }

    public Object getCriteria() {
        return criteria;
    }

    public void setCriteria(Object criteria) {
        this.criteria = criteria;
    }
}
